android/app/src/main/res/layout/item_twitter_feed.xml

```diff
@@ -3,6 +3,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@color/feed_bg"
+    android:paddingBottom="@dimen/feed_item_padding_top_bottom"
```

Comment:
Don't we have common spacing values?

Reply:
Do you mean the padding_medium, padding_small etc those?

Reply:
Yes